packages/wallet/src/account.ts

```diff
@@ -262,8 +262,8 @@ export class Account extends Signer {
 
   // publishConfig will publish the wallet config to the network via the relayer. Publishing
   // the config will also store the entire object of signers.
-  publishConfig(): Promise<TransactionResponse> {
-    return this.authWallet().wallet.publishConfig()
+  publishConfig(indexed?: boolean): Promise<TransactionResponse> {
```

Comment:
I noticed the argument name is `index` in the updateConfig method, lets keep these consistent. I think the name `indexed` is a bit more clear, so shall we update all to be consistent?